install.cmd

```diff
@@ -0,0 +1,3 @@
+@set PATH=%ProgramFiles(x86)%\Microsoft Visual Studio 14.0\Common7\IDE;%PATH%
+vsixinstaller /q /u:"c3d3dc68-c977-411f-b3e8-03b0dccf7dfc"
+vsixinstaller "%cd%\build\Debug\GitHub.VisualStudio.vsix"
```

Comment:
How about having some optional arguments to set which VS version to install to, and whether to install the debug or release build, so this can be useful for more scenarios than just specifically installing a debug build in 2015? That could be handier for automating things.